app/routes/participate/survey/consent.js

```diff
@@ -0,0 +1,19 @@
+import Ember from 'ember';
+import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
+import ENV from 'isp/config/environment';
+
+
+export default Ember.Route.extend(AuthenticatedRouteMixin, {
+    model() {
+        var session = this.modelFor('participate.survey');
+        // If participant has previously visited the survey (which requires granting consent),
+        // go directly to survey and skip consent page
+        if (ENV.featureFlags.continueSession) {
+            var frameIndex = session.get('frameIndex');
+            if (frameIndex && frameIndex >= 0) {
+                this.transitionTo('participate.survey');
```

Comment:
I may be misreading this- but if we can't save the consent property, then how do we know they ever consented?